.github/actions/verify-submodule-pull-request/action.yml

```
@@ -25,10 +25,16 @@ runs:
         New-Item -ItemType File -Force $Profile | Out-Null
         Get-Content -Raw ${{ github.action_path }}/functions.ps1 >> $Profile
 
+    # We are using env for the title input in these steps because it realistically could contain 
+    # quotes apostrophes or even both. This way those don't need to be escaped.
```

I don't know if we'll find tooling for that, but let's wrap comments at 120 chars here, too:
```suggestion
    # We are using env for the title input in these steps because it realistically could contain quotes, apostrophes or
    # even both. This way those don't need to be escaped.
```

---

That comma I snuck in hid itself from view 😁